spec/features/profile/edit_spec.rb

```diff
@@ -49,5 +49,18 @@
       expect(page).to have_content("Your password and confirm password do not match.")
 
     end
+    it "I cannot have a blank password" do
+
+      visit '/profile'
+      click_link 'Change password'
+
+      fill_in 'password', with:''
+      fill_in 'confirmation', with: ''
+      click_on 'Submit'
+
+
+      expect(page).to have_content("Your password is blank.")
```

Comment:
Was this still an error after removing the validate_presence_of :password? Does bcrypt not handle empty passwords?

Reply:
This was left in order to get 100% coverage

Reply:
Glad we did this